app/ai/voice/agents/breeze_buddy/agent/turn_strategies/config.py

```
@@ -0,0 +1,265 @@
+                raise ValueError("turn_analyzer strategy requires 'analyzer' parameter")
+
+            analyzer = v["analyzer"]
+            if isinstance(analyzer, str):
+                try:
+                    TurnAnalyzerType(analyzer)
+                except ValueError:
+                    valid_types = [t.value for t in TurnAnalyzerType]
+                    raise ValueError(
+                        f"Invalid analyzer type '{analyzer}'. "
+                        f"Must be one of: {', '.join(valid_types)}"
+                    )
```

Ruff (0.14.14)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)